### DISCUSSION TITLE
父项目src/filters/config-dev.properties中的配置为什么可以被解析进子项目

### DISCUSSION THREAD

**Discussion by xjcxjcxjc** [zuihou/zuihou-admin-cloud#293] [01.常见问题]
父项目中的src/filters/config-dev.properties配置了nacosip等参数，子项目使用${NACOS_PORT:@nacos.port@}获取配置参数，我想这真的可以获取到吗，我不太理解其中的联系

**Comment by zuihou** [comment]
先看文档

https://tangyh.top/doc/development/%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6.html#config-dev-properties
